### DISCUSSION TITLE
https://greendolphindance.com/2023/10/06/%E5%A1%94%E7%BD%97%E7%89%8C%E4%B8%BA%E4%BB%80%E4%B9%88%E4%B8%8D%E4%BC%9A%E5%87%BA%E9%94%99/

### DISCUSSION THREAD

**Discussion by giscus** [greendolphindance/greendolphindance.github.io#8] [General]
# https://greendolphindance.com/2023/10/06/%E5%A1%94%E7%BD%97%E7%89%8C%E4%B8%BA%E4%BB%80%E4%B9%88%E4%B8%8D%E4%BC%9A%E5%87%BA%E9%94%99/

Back to the rough ground!

https://greendolphindance.com/2023/10/06/%E5%A1%94%E7%BD%97%E7%89%8C%E4%B8%BA%E4%BB%80%E4%B9%88%E4%B8%8D%E4%BC%9A%E5%87%BA%E9%94%99/

**Comment by greendolphindance** [comment]
测试所有者发送的评论是否有绿色边框

**Reply by greendolphindance** [reply]
还真有

**Comment by hairkeeperonline** [comment]
所以大概类似“硬币抛出去的那一刻就知道内心想要的选项了”这种吗？

**Reply by greendolphindance** [reply]
不好说！总之是一些神秘体验
